lib/cloud_controller/config_schemas/base/api_schema.rb

```diff
@@ -311,8 +311,11 @@ class ApiSchema < VCAP::Config
               optional(:diego_sync) => { timeout_in_seconds: Integer },
             },
 
-            perm: {
-              enabled: bool,
+            # perm settings no longer have any effect but are preserved here
+            # for the time being to avoid breaking users as the perm
+            # setting was once required.
+            optional(:perm) => {
+              optional(:enabled) => bool,
```

Comment:
@mkocher so passing in `perm` as a config option will break the api_schema if `perm` isn't at least defined as optional here?

Am I right in thinking this config option ultimately comes from the bosh release? If so, why remove the `perm` property in `config/cloud-controller.yaml` but not here.

Reply:
Yes, you're right in thinking passing in perm config will break things if it's not defined as optional.

This usually comes from the bosh release and we could delete it there at the same time, but I'd prefer to not tie this change to an exact version of capi-release. I'm also suspecting that the various k8s cf projects might not use the bosh config.

I removed it from config/cloud-controller.yaml as I think of that as the canonical sample config for cloud controller, so it seems right to update it to not have any perm settings as that's what we'd hope all users will do eventually.